src/celeritas/ext/detail/GeantPhysicsList.cc

```diff
@@ -128,31 +130,47 @@ void GeantPhysicsList::add_gamma_processes()
     auto* physics_list = G4PhysicsListHelper::GetPhysicsListHelper();
     auto* gamma = G4Gamma::Gamma();
 
+    // Option to create GammaGeneral for performance/robustness
```

Comment:
Although the lambda function is straightforward to read, I think it's worth adding a note before or after the table in the documentation of `add_gamma_process()` mentioning that we either load individual processes or gamma general, which represents all processes in the table.